backend/Sender.ts

```diff
@@ -77,21 +76,21 @@ class Sender {
     }
 
     const sortedWordCounts: Map<string, number> = new Map([...wordCounts.entries()].sort((a, b) => b[1] - a[1]));
-    return sortedWordCounts;
+    await DatabaseMessageUtils.insertTopWords(sortedWordCounts);
   }
 
   static async sendTopWords(res: Response<any, number>): Promise<void> {
     if (res) {
-      log.debug('Sending top words to', {ip: res.req?.ip})
-      const sortedWordCounts = await this.getTopWords();
+      log.debug('Sending top words to', { ip: res.req?.ip });
+      const sortedWordCounts = await DatabaseMessageUtils.getTopWords();
       res.write('event: topWords\n');
-      res.write(`data: ${JSON.stringify({ topWords: Array.from(sortedWordCounts.entries()).slice(0, 20) })}`);
```

Comment:
Is the slice to limit the number of words no longer needed?

Reply:
The limit is set in the database select statement so slice isn't needed and the DB returns an Array anyways so Array.from() is not needed.